docs/api.rst

```diff
@@ -15,7 +15,7 @@ Errors
 
 All API endpoints use a standard error message format for any requests that return an HTTP status indicating an error (any 4xx or 5xx statuses):
 
-.. sourcecode:: http
+.. code:: html
```

Comment:
Should the `http` be changed to `html` everywhere?

Reply:
@apurvam good catch - this is a typo (should be http)